app/views/formatted-array.html

```diff
@@ -0,0 +1,13 @@
+<div class="formatted-array">
```

Comment:
I would suggest to just add `-template` or `-directive`
I think changing the folder would require a lot of change in the grunt build